docs/roadmap.rst

```diff
@@ -0,0 +1,66 @@
+##########################################
+Rubin Science Platform Status and Roadmap
+##########################################
+
+What can you do with the Rubin Science Platform?
+================================================
+
+Below you will find a detailed (but not exhaustive) list of current and future RSP capabilities.
+
+Please not the difference between Data Previews (DP0, DP1, DP2) which are early access programs before the start of the survey, and Data Releases (DR1, DR2 etc) which are the official data releases after the survey starts.
+For the planned dates for those events please consult project documentation and announcements.
+
+But before we get there, bear in mind there are three constraints to what you can do on the RSP:
+
+1. Scope. The Rubin Science Platform is aimed at providing access to Rubin Data Products and next-to-the-data analytics; there are some things it is not scoped to provide, for example large-scale re-processing of raw data (there are other inititatives that might help with that, but it's not a function of the RSP and is not planned to be).
+
+2. Affordability. There are limitations placed not by the capabilities of the RSP, but the budgeted cost to provide them to a very large userbase. For example, the storage per user is set by what the project can afford to provide. This could change with funding, change in the underlying cost of computing infrastructure or usage (for example if we have fewer users that we modeled there would be more resources for those users); but at any given time, it places a strong constraint on what can be provided by the project to everyone in the data-rights holding community.
+
+3. Developmental stage. Because the Rubin project predated the Science Platform era and many of the technologies we have since built on, there are actually significantly more developer-years ahead of us than behind us. We have a vigorous development program for the duration of the survey that gives us room to respond to needs beyond our construction requirements. However at this time, the RSP is in preview mode; that means there are many capabilities that will be available by DR1 that are not available in Data Preview mode, either because they are still under development or because they have not yet been demonstrated to scale sufficiently well.
```

Comment:
```suggestion
3. Developmental stage. At this time, the RSP is in preview mode; that means there are many capabilities that will be available by DR1 that are not yet available, either because they are still under development or because they have not yet been demonstrated to scale sufficiently well.
In fact, we expect that there are significantly more developer-years ahead of us than behind us. We have a vigorous development program for the duration of the survey that gives us room to respond to needs beyond our construction requirements.
```